ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -231,28 +236,38 @@ export const Homepage = withUserProfile()(class extends React.Component<Props, S
     });
   }
 
-  openVideo(videoLink: string): void {
-    this.setState({videoOpen: true, videoLink: videoLink});
+  openVideo(videoId: string): void {
+    this.setState({videoOpen: true, videoId: videoId});
   }
 
 
   render() {
     const {betaAccessGranted, videoOpen, accessTasksLoaded, accessTasksRemaining,
       eraCommonsError, eraCommonsLinked, eraCommonsLoading, firstVisitTraining,
-      trainingCompleted, quickTour, videoLink, twoFactorAuthCompleted,
-      dataUserCodeOfConductCompleted
+      trainingCompleted, quickTour, videoId, twoFactorAuthCompleted,
+      dataUserCodeOfConductCompleted, quickTourResourceOffset
     } = this.state;
+    const limit = this.quickTourResourcesDiv ? Math.floor(this.quickTourResourcesDiv.offsetWidth * 0.97 / 276) : 5;
```

Comment:
Lots of magic numbers here.

Reply:
Yeah, meant to add a comment with some context before pushing. Will add one shortly.

Reply:
Added a comment with an explanation and also got rid of one magic number since I'm measuring a different element (the 0.97 was accounting for a 3% margin but not needed now).